pkg/ucp/integrationtests/planes/testdata/awsplane_updated_v20231001preview_requestbody.json

```diff
@@ -1,7 +1,8 @@
 {
-    "location": "global",
```

Comment:
Lots of these examples had mistakes, or were just missing things that should be there. I did a pass and make sure we were consistent in how we're testing.

Most of UCP testing is done in integration tests since it's a single "unit".